recipes/beacon2-ri-tools/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "beacon2-ri-tools" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "ef35e9a9c2f059c8c442000e6371e872559b7cfa2511216e21c213f56083bb7b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/EGA-archive/beacon2-ri-tools/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  host:
+    - perl =>5.32
+    - perl-app-cpanminus
+    - perl-text-unidecode
+    - perl-json-xs
+    - perl-path-tiny
+    - perl-file-which
+    - perl-yaml-libyaml
+    - json_schema_validator
+    - perl-list-moreutils
+    - perl-text-csv_xs
+    - perl-minion-backend-sqlite
+    - perl-data-structure-util
+    - perl-mojolicious
+    - perl-minion
+    - perl-data-dumper
+    - perl-autodie
+    - perl-excel-writer-xlsx
+    - perl-encode
+    - perl-date-calc
+    - perl-app-cpanminus
+    - perl-perldoc
+    - perl-perlio-gzip
+    - perl-json-validator
+  run:
+    - perl =>5.32
+    - perl-app-cpanminus
+    - perl-text-unidecode
+    - perl-json-xs
+    - perl-path-tiny
+    - perl-file-which
+    - perl-yaml-libyaml
+    - json_schema_validator
+    - perl-list-moreutils
+    - perl-text-csv_xs
+    - perl-minion-backend-sqlite
+    - perl-data-structure-util
+    - perl-mojolicious
+    - perl-minion
+    - perl-data-dumper
+    - perl-autodie
+    - perl-excel-writer-xlsx
+    - perl-encode
+    - perl-date-calc
+    - perl-app-cpanminus
+    - perl-perldoc
+    - perl-json-validator
+test:
+  # Perl 'use' tests 
+  commands:
+    - beacon -h
+
+about:
+  home: https://github.com/EGA-archive/beacon2-ri-tools/tree/main
+  license: GPL-3.0-only
+  summary: "Script to parse a VCF having SnepEff/SnpSift annotations"
```

Comment:
Is that a correct summary?

Reply:
It sounds like overkill. I'm not sure why someone writes this in perl nowadays. Lets get started with the VCF mode and ask the beacon overlords if this is going to stay and we are supposed to carry those perl libs around forever.

Reply:
I will leave MongoDB for later then. 

about the description how is "Beacon v2 Reference Implementation (Data ingestion tools)" sound?